test/common/performance/aggregator/aggregator.go

```diff
@@ -113,9 +113,11 @@ func NewAggregator(benchmarkKey, benchmarkName, listenAddr string, expectRecords
 func (ag *Aggregator) Run(ctx context.Context) {
 	log.Printf("Configuring Mako")
 
+	makoClientCtx, _ := context.WithTimeout(ctx, time.Minute*10)
+
 	// Use the benchmark key created
 	// TODO support to check benchmark key for dev or prod
-	client, err := mako.SetupWithBenchmarkConfig(ctx, &ag.benchmarkKey, &ag.benchmarkName, ag.makoTags...)
+	client, err := mako.SetupWithBenchmarkConfig(makoClientCtx, &ag.benchmarkKey, &ag.benchmarkName, ag.makoTags...)
```

Comment:
Can we switch to using the new ConfigMaps here instead of `SetupWithBenchmarkConfig`? I'm fine with doing it in a different PR too.

Reply:
Let's do it in this one :) We first need https://github.com/knative/pkg/pull/868 merged

Reply:
Let's do in a different one so we fix all yamls too, can you open a ticket for it and assign to me?

Reply:
https://github.com/knative/eventing/pull/2202